src/stores/useHoloBusinessServiceStore.js

```diff
@@ -0,0 +1,35 @@
+import { defineStore } from 'pinia'
+import { authenticateAgent } from "../services/hbs"
+
+const kycLevel1 = 'holo_kyc_1'
+const kycLevel2 = 'holo_kyc_2'
+
+const useHoloBusinessServiceStore = defineStore('hbs', {
+  state: () => ({
+    agentKeyLevel: null
+  }),
+  actions: {
+    async loadAgentKycLevel(email, public_key) {
+
+      const authResult = await authenticateAgent(email, public_key)
+      if( authResult && authResult.id ) {
```

Comment:
Should we also check for `authResult.kyc` before doing a switch case on it?  Is it ever possible for the `authResult` to return without the `kyc` field?

Reply:
fixed.